GetIntoTeachingApi/Controllers/GetIntoTeaching/MailingListController.cs

```diff
@@ -82,6 +82,8 @@ public class MailingListController : ControllerBase
             [FromRoute, SwaggerParameter("Access token (PIN code).", Required = true)] string accessToken,
             [FromBody, SwaggerRequestBody("Candidate access token request (must match an existing candidate).", Required = true)] ExistingCandidateRequest request)
         {
+            request.Reference ??= User.Identity.Name;
```

Comment:
Could we not set the transaction name here `request.Reference = "Mailing list"`? Same question for the other transaction specific actions.

Reply:
Yeah that's going to be phase two if we find that we need that granularity as it will need a bit of work in the clients to ensure we're sending the transaction-level reference on the create access token request; its going to be easier to dashboard it in Grafana if its all at the service-level for now